src/newrelic.ts

```diff
@@ -64,10 +64,12 @@ function endSegment(spec: Spec) {
 function preload(this: any, opts: any) {
   const seneca = this;
   const { options }: { options: NewRelicOptions } = opts;
-  const segmentIsEnabled = options && options.segment && options.segment.enabled;
-  const tracingIsEnabled = options && options.tracing && options.tracing.enabled;
-  const metricsIsEnabled = options && options.metrics && options.metrics.enabled;
-  const eventsIsEnabled = options && options.events && options.events.enabled;
+  const isPluginEnabled = options && options.enabled
+
+  const segmentIsEnabled = isPluginEnabled && options.segment && options.segment.enabled;
```

Comment:
I would recommend a simpler alternative. Instead of having:
```javascript
const segmentIsEnabled = isPluginEnabled && ...;
const tracingIsEnabled = isPluginEnabled && ...;
const metricsIsEnabled = isPluginEnabled && ...;
const eventsIsEnabled = isPluginEnabled && ...;
```

just use the approach you used in `function newrelic`:
```javascript
const isPluginEnabled = options && options.enabled

if (!isPluginEnabled) {
  return
}

// rest of the code
//
```